olive/common/hf/peft.py

```diff
@@ -28,15 +28,13 @@ def make_export_compatible_peft(model: torch.nn.Module, merge_weights: bool = Fa
         - Rescale the lora_B weights with scaling factor, change scaling factor to 1 (int) so that it doesn't appear
           in the exported model (only works for torchscript export)
     """
-    if not is_peft_model(model):
-        return model
-
     if merge_weights:
```

Comment:
can you add this inside the condition at line 36? merge_weights is only valid for peft models.